app/connectors/CustomsDeclareExportsConnector.scala

```diff
@@ -37,27 +38,38 @@ import scala.concurrent.{ExecutionContext, Future}
 class CustomsDeclareExportsConnector @Inject()(appConfig: AppConfig, httpClient: HttpClient) {
 
   private val logger = Logger(this.getClass)
+  
+  private def logPayload[T](prefix: String, t: T)(implicit wts: Writes[T]): T = {
```

Comment:
can you change `t` to `payload`? I know that this variable has really small scope, but it's easier to read specific naming